src/components/DataTypesMenu.vue

```diff
@@ -2,25 +2,31 @@
   <popover :visible="visible" :align="alignLeft" bottom @closed="close">
     <div class="data-types-menu__body">
       <div class="data-types-menu__section">
-        <div class="data-types-menu__option" @click="createConvertStep('integer')">
+        <div class="data-types-menu__option--active" @click="createConvertStep('integer')">
           <span class="data-types-menu__icon">123</span>
           <span>Integer</span>
         </div>
-        <div class="data-types-menu__option" @click="createConvertStep('float')">
+        <div class="data-types-menu__option--active" @click="createConvertStep('float')">
           <span class="data-types-menu__icon">1.2</span>
           <span>Float</span>
         </div>
-        <div class="data-types-menu__option" @click="createConvertStep('text')">
+        <div class="data-types-menu__option--active" @click="createConvertStep('text')">
           <span class="data-types-menu__icon">ABC</span>
           <span>Text</span>
         </div>
-        <div class="data-types-menu__option" @click="createConvertStep('date')">
+        <div
+          :class="{
+            'data-types-menu__option--active': columnTypes[columnName] === 'string',
+            'data-types-menu__option--deactivated': !(columnTypes[columnName] === 'string'),
+          }"
```

Comment:
```suggestion
          }"
          title="Only string column can be converted to date"
```

Reply:
This will display a small native html tooltip with the content of "title"

Reply:
Nice, thanks !